crates/node/builder/src/setup.rs

```diff
@@ -64,6 +68,7 @@ where
         static_file_producer,
         evm_config,
         exex_manager_handle,
+        None,
```

Comment:
Here would be a good place to specify optional `EraImportSource`. It could be delegated to the function argument or created on the spot based on a flag or simply hardcoded. How do we obtain / create it?

Reply:
I believe we first derive this from the cli args during launch, and eventually need to make this default based on the chainid, also during launch, when we create the pipeline:


https://github.com/paradigmxyz/reth/blob/22452e8ace68ed19ebe7110d2cd34a94617c8ec5/crates/node/builder/src/launch/engine.rs#L156-L156